ext/src/Smallint.c

```diff
@@ -242,10 +242,6 @@ PHP_METHOD(Smallint, div)
     }
 
     result->data.smallint.value = self->data.smallint.value / smallint->data.smallint.value;
-    if (result->data.smallint.value * smallint->data.smallint.value != self->data.smallint.value) {
```

Comment:
I'm curious what this is for... Is there any combination of values that would make this true?